vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/RoundExpressionProcessor.java

```diff
@@ -38,8 +38,8 @@
     {
         String roundingModes = Stream.of(RoundingMode.values()).map(e -> e.name().toLowerCase())
                 .collect(Collectors.joining("|"));
-        String pattern = String.format("^round(?:\\((-?\\d+(?:\\.\\d*)?)(?:,\\s*(\\d+))?(?:,\\s*(%s))?\\))$",
-            roundingModes);
+        String pattern = String.format("^round(?:\\((-?\\d+(?:\\.\\d*)?[Ee]?-?\\+?\\d*)"
```

Comment:
this regex will also match the following invalid strings:
 - `0.15237E`
 - `0.15237E-+2`
 - `0.15237-2`
 - `0.15237+2`

Reply:
fixed

Reply:
I believe all items from the comment above should be added to negative unit tests

Reply:
added